src/Drawing/BasicDrawing.cpp

```diff
@@ -52,17 +53,19 @@ void BasicDrawing::DrawText(const std::string text, const LVector2& point, const
     ::DrawText(text.data(), point.X, point.Y, size, c);
 }
 
-void BasicDrawing::DrawText(const std::string text, const LVector2& point, float size, float spacing, const LColor& color) {
+void BasicDrawing::DrawText(const std::string text, const LVector2& point, float size, float spacing, const LColor& color, const float rotation) {
     if (!CanDraw()) return;
     RColor c = ConvertEngineColor(color);
-    ::DrawTextEx(::GetFontDefault(), text.data(), ::Vector2({point.X, point.Y}), size, spacing, c);
+    ::DrawTextPro(::GetFontDefault(), text.data(), ::Vector2({point.X, point.Y}), 
```

Comment:
Might be outside the scope of this commit, but why not just have a implicit cast defined in LVector2 that converts it into a Vector2 so that you don't have to always do explicit casts for stuff like this?